packages/currencies/src/reactNative.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import * as icons from "./data/icons/reactNative";
+
+const iconsByCoinType = {
+  "0": icons.bitcoin,
+  "1": icons.bitcoin,
+  "2": icons.litecoin,
+  "3": icons.dogecoin,
+  "5": icons.dash
```

Comment:
we should unify some logic between both react and reactNative, basically should do:

coinTypeToIconName.js
```js
export default {
  "0": "bitcoin",
  "1": "bitcoin",
  "2": "litecoin",
}
```

and then on each react & reactNative.js

```js
export function getIconByCoinType(coinType: number): ?Icon {
  return icons[coinTypeToIconName[coinType]];
}
```